features/shipped/FxA-33-choose-what-to-sync/README.md

```diff
@@ -8,6 +8,21 @@ As a user, I want to choose what data types to sync as part
  of the account setup process, so that it's more obvious what
  functionality I'm signing up for.
 
+## Outcomes
+
+We first shipped "Choose What to Sync" (CWTS) in Q4 of 2015.
+This first release was limited to Firefox Nightly users only.
+
+We also completed work to support CWTS in future versions of Firefox for iOS v2.
+
+January 2016 - CWTS was enabled in Firefox for Android v44 stable release.
+
+February 2016 - CWTS is available for Firstrun users in Firefox Desktop.
```

Comment:
Did it have any noticeable impact on our metrics one way or another, and/or are we happy that it seems to have left them unimpacted?